documentation/ipi-install/modules/ipi-ipv6-install-network-requirements.adoc

```diff
@@ -3,9 +3,9 @@
 
 === SLAAC Addressing
 
-If you don't plan to use SLAAC addresses on your OpenShift node then SLAAC addressing should be disabled for the `baremetal` networks, that means that if your network equipment is configured to send SLAAC addresses when replying to Route Advertisements that behavior should be changed, so it only sends the route and not the SLAAC address.
+If you don't plan to use SLAAC footnote:[Stateless Address AutoConfiguration] addresses on your OpenShift node, then it should be disabled for `baremetal` networks, that means that if your network equipment is configured to send SLAAC addresses when replying to Route Advertisements that behavior should be changed, so it only sends the route and not the SLAAC address.
```

Comment:
TMK SLAAC doesn't work at all with OCP (so I've heard). This may need to change to be more reflective of that.

Reply:
@mvazquezc can you confirm that SLAAC doesn't work at all with OCP?

Reply:
We haven't tested a deployment with SLAAC addresses. What we can say is that if your nodes have SLAAC address  + BM network IPv6 address, the Kubelet will start with the SLAAC address and the deployment will fail. If your network is capable of routing SLAAC addresses for the BM network maybe it works (we haven't tested that).